examples/knative/go/kn-go-echo/server.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+)
+
+func main() {
+	ctx := context.Background()
+	protocol, err := cloudevents.NewHTTP()
```

Comment:
You could simplify this with `NewClientHTTP` (godoc: https://pkg.go.dev/github.com/cloudevents/sdk-go/v2@v2.4.1/client#NewHTTP)

Reply:
That's much better. I'll make that change.